src/app/shared/editor/editor.component.ts

```diff
@@ -36,6 +37,7 @@ export class EditorComponent implements OnInit {
     toolbar:
       'undo redo | bold italic underline strikethrough | \
       bullist numlist | link',
+    elementpath: false,
```

Comment:
This removes the circled element:
<img width="658" alt="Screenshot 2022-07-29 at 15 02 47" src="https://user-images.githubusercontent.com/6022408/181766233-b2de9f68-e090-4aca-b37c-bf4e63a317ec.png">